core/services/keeper/validate_test.go

```diff
@@ -1,12 +1,10 @@
 package keeper
 
 import (
+	"github.com/smartcontractkit/chainlink/core/testdata/testspecs"
+	"github.com/stretchr/testify/require"
 	"testing"
 	"time"
```

Comment:
nit put i think your sorting is off here. should be able to run the import formatter to fix it

Reply:
Hmm, are you sure? In GoLand they are displayed as being sorted and no formatting change is recommended.

Reply:
Sorry, I was using the `gofmt` settings in my IDE which is the default setting instead of `goimports`. I changed it now and it reordered them as you suggested :)